prime-angular-frontend/src/app/modules/enrolment/shared/components/college-certification-form/college-certification-form.component.ts

```diff
@@ -96,7 +96,7 @@ export class CollegeCertificationFormComponent implements OnInit {
   }
 
   private setValidations() {
-    this.formUtilsService.setValidators(this.licenseNumber, [Validators.required, FormControlValidators.requiredLength(5)]);
+    this.formUtilsService.setValidators(this.licenseNumber, [Validators.required, this.alphaNumeric()]);
     this.formUtilsService.setValidators(this.licenseCode, [Validators.required]);
```

Comment:
Great validator!  You'll want to move this into FormControlValidators so it can be reused within the application.